lib/iris/tests/unit/experimental/um/test_FieldsFileVariant.py

```diff
@@ -77,6 +77,27 @@ def test(self):
 
 
 @tests.skip_data
+class Test_packing_exceptions(tests.IrisTest):
```

Comment:
I really like these tests. Could you consolidate these with the work in #1639 (https://github.com/SciTools/iris/pull/1639/files#diff-842b0e426cc7ad703bdbd68c2b357830R351)